.github/workflows/ci.yaml

```diff
@@ -0,0 +1,131 @@
+# Code generated by dhall-to-yaml.  DO NOT EDIT.
+jobs:
+  checkPipeline:
+    name: render-CI-pipeline
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: 'actions/checkout@v2'
+      - name: Install asdf
+        uses: 'asdf-vm/actions/setup@v1.0.1'
+      - name: Install asdf plugins
+        run: './scripts/asdf-add-plugins.sh'
+      - name: Install asdf tool versions and packages
+        run: './ci/asdf-install.sh'
+      - name: Check that the CI pipeline definition is up-to-date with the dhall configuration
+        run: ci/check-rendered-pipeline-up-to-date.sh
+  dhallCheck:
```

Comment:
there's no dhall in this repo, do you still want this here?

Reply:
There is - it's used to generate the CI pipeline.